daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-overview.md

```diff
@@ -8,44 +8,30 @@ description: "Overview of the Dapr Pub/Sub building block"
 
-Similarly, a **consumer** will receive messages from a topic without knowledge of what producer sent it. This pattern is especially useful when you need to decouple microservices from one another.
-
-Dapr provides a publish/subscribe API that provides at-least-once guarantees and integrates with various message brokers implementations. These implementations are pluggable, and developed outside of the Dapr runtime in [components-contrib](https://github.com/dapr/components-contrib/tree/master/pubsub).
+The publish/subscribe API in Dapr provides at-least-once guarantee and integrates with various message brokers and queing systems. The specific implementation to your application is pluggable and configured externally at runtime. This approach removes dependancy from your application and, as a result, makes your application more portable. The complete list of available publish/subscribe implementations is available in [components-contrib](https://github.com/dapr/components-contrib/tree/master/pubsub) repository.
 
 ## Features
 
 ### Publish/Subscribe API
 
-The API for Publish/Subscribe can be found in the [spec repo]({{< ref pubsub_api.md >}}).
-
-### At-Least-Once guarantee
-
-Dapr guarantees At-Least-Once semantics for message delivery.
-That means that when an application publishes a message to a topic using the Publish/Subscribe API, it can assume the message is delivered at least once to any subscriber when the response status code from that endpoint is `200`, or returns no error if using the gRPC client.
-
-### Consumer groups and multiple instances
-
-The burden of dealing with concepts like consumer groups and multiple instances inside consumer groups is all handled autoamtically by Dapr:
-
-* When multiple instances of the same application (same IDs) subscribe to a topic, Dapr will deliver each message to only one instance of that application. 
-* If two different applications (different IDs) subscribe to the same topic, Dapr will deliver each message to only one instance of each application.
+The publish/subscribe API is located in the [spec repo]({{< ref pubsub_api.md >}}).
 
-### Cloud events
+### Message Format
 
-Dapr follows the [CloudEvents 1.0 Spec](https://github.com/cloudevents/spec/tree/v1.0) and wraps any payload sent to a topic inside a Cloud Events envelope, using `Content-Type` header value for `datacontenttype` attribute.
+To enable message routing and to provide additonal context with each message Dapr uses the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/tree/v1.0) as its message format. Any message sent by an paylication to a topic using Dapr will autoamtically be "wrapped" in Cloud Events envelope, using `Content-Type` header value for `datacontenttype` attribute.
```

Comment:
Any message sent by an application to a topic using Dapr will automatically be "wrapped" .....